src/ess/beer/conversions.py

```diff
@@ -112,8 +115,20 @@ def _compute_d(
     return d
 
 
-def _tof_from_dhkl(
+def time_of_arrival(
     event_time_offset: sc.Variable,
+    tc: sc.Variable,
+):
+    '''Does frame unwrapping based on the cutoff time `tc` and `event_time_ffset`.
+    Events before `tc` are assumed to come from the previous pulse.'''
```

Comment:
Some general comments on docstring style:
- According to https://peps.python.org/pep-0257/ they should always use double quotes.
- Sphinx will use the entire first paragraph as the summary which is displayed in the overview tables:
<img width="917" height="310" alt="summary" src="https://github.com/user-attachments/assets/75fe3403-522a-459c-b657-35ee094b005b" />
These get unwieldy when the summary is long. So as a general rule, it is best to limit the summary to a single line and then use a separate paragraph for the detailed description. See also the [coding convention](https://scipp.github.io/development/coding-conventions.html#docstrings)